src/yardstick/cli/config.py

```diff
@@ -68,6 +68,9 @@ class ResultSet:
     declared: list[artifact.ScanRequest] = field(default_factory=list)
     matrix: ScanMatrix = field(default_factory=ScanMatrix)
 
+    def images(self) -> list[str]:
```

Comment:
I don't understand this change. Is this used somewhere else in the PR?

Reply:
it's being used in grype-db, but I can add the helper there (and remove it here)

Reply:
Thanks! I didn't realize it was considered part of the exported library API.